sdk/core/core-tracing/lib/index.ts

```diff
@@ -33,3 +32,39 @@ export { TimedEvent } from "./interfaces/TimedEvent";
 export { TraceFlags } from "./interfaces/trace_flags";
 export { TraceState } from "./interfaces/trace_state";
 export { Tracer } from "./interfaces/tracer";
+
+// OpenCensus Interfaces
+export {
+  Tracer as OpenCensusTracer,
+  Span as OpenCensusSpan,
+  TracerBase as OpenCensusTracerBase,
+  Func as OpenCensusFunc,
+  Link as OpenCensusLink,
+  Annotation as OpenCensusAnnotation,
+  Attributes as OpenCensusAttributes,
+  CanonicalCode as OpenCensusCanonicalCode,
+  MessageEvent as OpenCensusMessageEvent,
+  SpanOptions as OpenCensusSpanOptions,
+  SpanContext as OpenCensusSpanContext,
+  TraceOptions as OpenCensusTraceOptions,
+  LinkType as OpenCensusLinkType,
+  MessageEventType as OpenCensusMessageEventType,
+  SpanKind as OpenCensusSpanKind,
+  Status as OpenCensusStatus,
+  SpanEventListener as OpenCensusSpanEventListener,
+  TraceState as OpenCensusTraceState
+} from "./interfaces/OpenCensus/model";
```

Comment:
The reason we added interfaces for OpenTelemetry was because the library wasnt shipped yet.
What is the reason to export OpenCensus types from our core library? I would expect typings for OpenCensus library to already exist

Reply:
Yeah I think I was being silly here. I'm going to try to import the package and see what that does for us.